scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -819,12 +819,24 @@ def run(self):
                                 if kind == "Pod":
                                     extra_fields["pod_name"] = name
                                     extra_fields["pod_namespace"] = namespace
-                                    pod = k8s_cache.pod(
-                                        namespace,
-                                        name,
-                                        current_time,
-                                        query_options=k8s_events_query_options,
-                                    )
+                                    try:
+                                        pod = k8s_cache.pod(
+                                            namespace,
+                                            name,
+                                            current_time,
+                                            query_options=k8s_events_query_options,
+                                        )
+                                    except k8s_utils.K8sApiNotFoundException as e:
```

Comment:
Change looks reasonable to me as long as we don't potentially mask some other errors which should actually be logged under info.

Reply:
Also, do you perhaps have access to some error message which arises in such situation?

Just wondering if we can scope this except some more based on the error message..

Reply:
```K8sApiNotFoundException: The resource at location `/api/v1/namespaces/scalyr/pods/<pod name>` was not found```

Not sure if there is any meaningful way we can scope this down further, since this exception is about a queried resource being missing, and the `try` is around the code where we attempt to query a pod and nothing else.